CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # [Versions](https://mui.com/versions/)
 
+## 6.3.0
+
+<!-- generated comparing v6.2.1..master -->
+
+_Dec 23, 2024_
+
+A big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- Fix invalid HTML structure in the Accordion component (#44408) @ZeeshanTamboli
```

Comment:
Should we label this as "Minor version changes"?

```suggestion
### Minor version changes

- Fix invalid HTML structure in the Accordion component (#44408) @ZeeshanTamboli
```

Reply:
No, we never do this. We add separate heading only for breaking changes.